src/ctelnet.cpp

```diff
@@ -1340,6 +1332,16 @@ void cTelnet::processTelnetCommand(const string& command)
                 packageName.remove(QLatin1Char('\\'));
                 packageName.remove(QLatin1Char('.'));
 
+                if (mpHost->mServerGUI_Package_version != version) {
+                    postMessage(tr("[ INFO ]  - The server wants to upgrade the GUI to new version '%1'.\n"
+                                   "Uninstalling old version '%2'.")
+                                .arg(version, mpHost->mServerGUI_Package_version != QStringLiteral("-1") ? mpHost->mServerGUI_Package_version : QStringLiteral("(unknown)")));
```

Comment:
It is *a bit of an old chestnut* but in this comparison (to `"-1"`) and the one a couple of lines down (to `"nothing"`) we have already declared the same texts in the initialiser list with `QLatin1String` - given that the `QString::operator!=` operator is one that can handle `QLatin1String` directly (using a raw `char` comparison) we do not need to use a further 2 x 2 `QString`s for these comparisons even if we do generate them for free at runtime...

Reply:
@SlySven Would you file the chestnut as an issue of its own though? :bulb: It will get lost in the woods down here otherwise.

Reply:
imho it's not worth cluttering the code with most consts for this :(